client/app/domain/general-informations/GENERAL_INFORMATIONS.html

```diff
@@ -292,5 +292,30 @@
                 </div>
             </div>
         </div>
+        <div class="col-xm-12 col-md-4 mb-5">
+            <oui-tile
+                class="h-100"
+                data-heading="{{ ::'domain_dashboard_options' | translate }}"
+                data-loading="$ctrl.loading.options">
+                <oui-tile-definition data-term="{{ ::'domain_dashboard_options_dns_anycast' | translate }}">
+                    <oui-tile-description>
+                        <span data-ng-if="$ctrl.options.dnsAnycast"
+                              data-translate="{{ 'domain_dashboard_options_status_' + ($ctrl.options.dnsAnycast.optionActivated ? 'enabled' : 'disabled') }}">
+                        </span>
+                        <span data-ng-if="!$ctrl.options.dnsAnycast"
+                              data-translate="domain_dashboard_options_unavailable">
+                        </span>
+                    </oui-tile-description>
+                    <oui-action-menu data-compact data-align="end"
+                                     data-ng-if="$ctrl.options.dnsAnycast && $ctrl.options.dnsAnycast.optionActivated">
+                        <oui-action-menu-item data-text="{{ ::'domain_dashboard_options_desactivate_option' | translate }}"
+                            aria-label="{{ ::'domain_dashboard_options_desactivate_option' | translate }}"
+                            data-on-click="setAction('options/delete/domain-options-delete', { domain: $ctrl.domain, option: $ctrl.options.dnsAnycast })"
+                            data-ng-if="$ctrl.options.dnsAnycast.optionActivated">
```

Comment:
You test it on the parent element. Why retesting it here ?

Reply:
You are right. This redundant check has been removed from the menu-item. Thanks :)